CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Improved description of the `--control-plane-az` parameter when templating a cluster.
 
+### Fixed
+
+- Create cache directory if it does not exist, yet.
```

Comment:
Let's try and explain that so that every user can understand the change. I'll make an attempt and you can improve it, okay?

```suggestion
- Prevent kubectl-gs from checking the latest version on every execution, and use a cache instead.
```

Reply:
Hmm. Looks like that description doesn't really fit.